CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 # Copyright (c) 2015-2018 by Radovan Bast, Roberto Di Remigio, Jonas Juselius, and contributors.
 
 # set minimum cmake version
-cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.3 FATAL_ERROR)
 
 # project name
 project(MRCPP CXX)
```

Comment:
Consider using the extended syntax:
```
project(MRCPP LANGUAGES CXX VERSION ...)
```
This initialized all the `PROJECT` related options consistently. The latest Autocmake implementes the `LANGUAGES` option, so you should update to that one.